src/Sulu/Bundle/ContactBundle/Api/Contact.php

```diff
@@ -1036,7 +1036,7 @@ public function getMainAddress()
         if (!is_null($contactAddresses)) {
             /** @var ContactAddressEntity $contactAddress */
             foreach ($contactAddresses as $contactAddress) {
-                if ((bool) $contactAddress->getMain()) {
+                if ((bool)$contactAddress->getMain()) {
```

Comment:
This change causes the failure in CI, because we have configured it to have a space in that case.